symbolic-symcache/src/writer.rs

```diff
@@ -140,6 +146,8 @@ impl<'a> SymCacheConverter<'a> {
             self.process_symbolic_symbol(&symbol);
         }
 
+        self.is_windows_object = false;
+
```

Comment:
```suggestion
```

Should line 137 already cover this case? `matches!` will just return `false` for us.

Reply:
This was intentional. I only want to scope this to the call. Which is a bit leaky anywhere, as there is a bunch of `?` in that function that early-return and bypass this.

Though the `process_symbolic_function` is public so anyone could just process a random function as well, for which we don’t know where it came from.